packages/spruce-skill-server/tests/SpruceTest.js

```diff
@@ -56,7 +60,10 @@ module.exports = basePath => {
 						)
 					}
 					this.mocks[mock.key] = mock
-					await mock.setup(options)
+					await mock.setup({
+						...options,
+						sandbox
+					})
```

Comment:
@kengoldfarb can you provide an example in one of the tests on how this is used and extended?